spreadsheet/cell.go

```diff
@@ -65,6 +66,7 @@ func (c Cell) SetInlineString(s string) {
 	c.x.Is = sml.NewCT_Rst()
 	c.x.Is.T = unioffice.String(s)
 	c.x.TAttr = sml.ST_CellTypeInlineStr
+	setMaxColumn(c)
```

Comment:
If we do the search when MaxIndex is called, then we don't have to keep this up to date?  Seems like an easier option.

Reply:
fixed